alf/algorithms/dynamic_action_repeat_agent.py

```diff
@@ -253,6 +256,10 @@ def train_step(self, rl_exp: Experience, state: ActionRepeatState):
 
     def calc_loss(self, rl_experience, rl_info):
         """Calculate the loss for training ``self._rl``."""
+        with alf.summary.scope(self._name):
+            # summarize_reward will check ``_debug_summaries`` and
+            # ``should_record_summaries()``
+            self.summarize_reward("training_reward", rl_experience.reward)
```

Comment:
Why don't you want to check  ``_debug_summaries`` and ``should_record_summaries()``?
Even you don't do it here, ``should_record_summaries()`` will always be checked in summary_ops.py

Reply:
summarize_reward already does the check. So there is no need to add more code here.